opendevin/runtime/docker/ssh_box.py

```diff
@@ -200,7 +200,7 @@ class DockerSSHBox(Sandbox):
     def __init__(
         self,
         container_image: str | None = None,
-        timeout: int = config.sandbox_timeout,
+        timeout: int = config.sandbox.timeout,
```

Comment:
I believe that if we search the repo for `sandbox_timeout`, we can find other occurrences of this var, because `evaluation` is using this.